src/atc/schema_manager/schema_manager.py

```diff
@@ -0,0 +1,105 @@
+import copy
+from typing import Dict
+
+import pyspark.sql.types as T
+
+from atc.atc_exceptions import AtcException
+from atc.configurator import Configurator
+from atc.singleton import Singleton
+
+
+class NoSuchSchemaException(AtcException):
+    pass
+
+
+class FalseSchemaDefinitionException(AtcException):
+    pass
+
+
+class UnregisteredSchemaDefinitionException(AtcException):
+    pass
+
+
+class SchemaManager(metaclass=Singleton):
+
+    # This dict contains the registered schemas
+    _registered_schemas: Dict[str, T.StructType] = {}
+
+    def __init__(self):
+        pass
+
+    def register_schema(self, schema_name: str, schema: T.StructType):
+        self._registered_schemas[schema_name] = schema
+
+    def get_schema(self, schema_identifier: str):
```

Comment:
use a doc_string to describe that the handle is the same as configurator table_id